helm/content-fluentd/values.yaml

```diff
@@ -0,0 +1,14 @@
+service:
+  name: "" # The name of the service, should be defined in the specific app-configs folder.
+  namespace: kube-system
```

Comment:
Not sure if we should deploy in the kube-system namespace. Although I do agree we should have a single deployment/ kubernetes cluster, especially when we'll move to namespaces/stack or env. But probably we should have our own "system" namespace and let kube-system with the kubernetes system apps.

Reply:
I think the official k8s fluentd config puts it into a `system-logging` namespace, but I decided to place it in the `kube-system` rather than the apps space that we already have. 

Where should the template for a new namespace live? Or should this have its own namespace?